### PR TITLE
feat(route): add swjtu recruitment info rss

### PR DESCRIPTION
**Pull request by qizidog**
## 完整路由地址 / Example for the proposed route(s)

```routes
/swjtu/jyzpxx
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

增加 `西南交通大学-就业招聘信息` 路由，单次拉取含30条信息

### CODE REVIEW

File: lib/v2/swjtu/jyzpxx.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                        job_info.push(`<tr> 
+                        <td width="50">${job.zwmc}</td> 
+                        <td width="50">${job.yxmc}/月</td> 
+                        <td width="50">${job.gzdz}</td> 
+                        <td width="70">${job.zyyqmc}</td> 
+                        <td width="125">${job.zwms}</td> 
+                        </tr>`);
+                    }
+                }
+                job_info.push('</table></div>');
```
Comment (TonyRL) [inline lib/v2/swjtu/jyzpxx.js:46]: Please use [art-template](https://aui.github.io/art-template/docs/)
docs: https://docs.rsshub.app/joinus/script-standard.html#template

File: lib/v2/swjtu/jyzpxx.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                <p>行业名称：${item.hyyjmc}（${item.hyejmc}）</p>
+                <p>公司规模：${item.rsgmmc}</p>
+                <p>招聘职位：</p>
+                ${job_info.join('')}
+                <p>工作地点：${item.szxmc} ${item.xxdz}</p>
+                <p>招聘截止日期：${item.zpjzrq}</p>
+                <p>简历投递邮箱：${item.jltdyx}</p>
+                <p>单位网站：${item.dwwz}</p>
+                ${content.zpxxEditor}
+                `,
```
Comment (TonyRL) [inline lib/v2/swjtu/jyzpxx.js:64]: Please use [art-template](https://aui.github.io/art-template/docs/)
docs: https://docs.rsshub.app/joinus/script-standard.html#template

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/swjtu/jyzpxx.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootURL = 'https://jiuye.swjtu.edu.cn/career/';
+
+module.exports = async (ctx) => {
+    const resp = await got({
+        method: 'post',
+        url: `${rootURL}/zpxx/search/zpxx/1/30`,
```
Comment (TonyRL) [inline lib/v2/swjtu/jyzpxx.js:9]: `rootURL` ended with a slash, `url` also has a slash after `rootURL`. The final URL will contain **duplicated** slashes

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/swjtu/jyzpxx.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                `,
+                    link: `${rootURL}/zpxx/view/zpxx/${item.zpxxid}`,
+                };
+            });
+        })
+    );
+
+    ctx.state.data = {
+        title: '西南交大-就业招聘信息',
+        link: `${rootURL}/zpxx/zpxx`,
```
Comment (TonyRL) [inline lib/v2/swjtu/jyzpxx.js:73]: `rootURL` ends with a slash, `link` also contains a slash after `rootURL`. The final URL will have duplicated slashes

Comment (qizidog) [reply lib/v2/swjtu/jyzpxx.js:23]: The end slash of `rootURL` has been removed.

**Review by qizidog — COMMENTED** [review]
fix(route): use art-template

**Pull request merged**